lib/install.js

```diff
@@ -28,10 +28,9 @@ bin.run(args, err => {
 			.cmd(cfg)
 			.cmd('make install');
 
-		return builder.run(err => {
-			if (err) {
-				log.error(err.stack);
-				return;
+		return builder.run(buildErr => {
+			if (buildErr) {
+				throw buildErr;
```

Comment:
Maybe it's better to still log error but exits from process with error code?
```js
log.error(err.stack);
process.exit(1);
```

Reply:
As I understand it throwing the error will result in both of those things happening.

Reply:
Also, `process.exit(1)` violates `unicorn/no-process-exit` as you can see here: https://travis-ci.org/imagemin/jpegtran-bin/jobs/347744916

Reply:
throwing en error would not logs it properly
``log.error(err.stack);``
Maybe put loging back?